app/assets/stylesheets/application.css

```diff
@@ -13,10 +13,17 @@
  *= require_tree .
  *= require_self
  */
-
+.checkin_header a {
+  font-size: 30px;
+  color: white;
+  margin-right: 100px;
+}
+.checkin_header a:hover {
+  color: grey;
+}
```

Comment:
add a blank line between this two blocks, and add a space between your class name an your curly braces